Dockerfile

```diff
@@ -4,5 +4,8 @@ WORKDIR /usr/src/tarssh
 COPY . .
 
 RUN cargo install --path .
+RUN mkdir /var/empty
+RUN chown nobody:nogroup /var/empty
+RUN chmod u=rx,g=rx,o-rwx /var/empty
```

Comment:
`rx` should do, 0555?

Seems odd it's not included in the environment by default.